start-chrome-for-playr.sh

```diff
@@ -58,7 +144,21 @@ else
 	reload_url=file://$2
 fi
 
+# Only preference file adjustments are needed on Linux
```

Comment:
het feit dat dit niet in de MacOS file aanwezig was wil niet zeggen dat het niet nodg is, waarschijnlijk is het een tekortkoming in het macOS script